packages/xo-server/src/api/host.mjs

```diff
@@ -624,6 +628,20 @@ getSmartctlInformation.resolve = {
   host: ['id', 'host', 'view'],
 }
 
+export function getMdadmHealth({ host }) {
+  return this.getXapi(host).getHostMdadmHealth(host._xapiId, { cache: CACHE_MDADM })
+}
+
+getMdadmHealth.description = 'get mdadm health status'
```

Comment:
`get mdadm health status` sound like it returns a simple string..

Reply:
What about ```'retrieve the mdadm RAID health information' ``` ?